tools/make_release_tarball

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+if [ ! -e tools/make_release_tarball  -o  "$1" != "" ]
+then
+  echo "Handy script for creating tarball: Use from the checkout directory"
+  exit 0
+fi
+
+set -e
+
+VERSION_MAJOR=$(grep set.MIR_VERSION_MAJOR CMakeLists.txt | cut -f2 -d' ' | tr -d \))
+VERSION_MINOR=$(grep set.MIR_VERSION_MINOR CMakeLists.txt | cut -f2 -d' ' | tr -d \))
+VERSION_PATCH=$(grep set.MIR_VERSION_PATCH CMakeLists.txt | cut -f2 -d' ' | tr -d \))
+
+VERSIONED_NAME=mir-$VERSION_MAJOR.$VERSION_MINOR.$VERSION_PATCH
+
+SCRATCH_DIR=$(mktemp -d)
+BUILD_DIR=$(mktemp -d)
+INSTALL_DIR=$(mktemp -d)
+
+function cleanup() {
+	ARG=$?
+	[ -d $SCRATCH_DIR ] && rm -r $SCRATCH_DIR
+	[ -d $BUILD_DIR ] && rm -r $BUILD_DIR
+	[ -d $INSTALL_DIR ] && rm -r $INSTALL_DIR
+	exit $ARG
+}
+
+trap cleanup EXIT
+
+echo "Generating Mir tarball…"
+pushd tests/acceptance-tests/wayland/wlcs
+git archive --format=tar -o $SCRATCH_DIR/$VERSIONED_NAME-wlcs.tar --prefix=$VERSIONED_NAME/tests/acceptance-tests/wayland/wlcs/ HEAD
+popd
+git archive --format=tar -o $SCRATCH_DIR/$VERSIONED_NAME.tar --prefix=$VERSIONED_NAME/ HEAD
+tar --concatenate -vf $SCRATCH_DIR/$VERSIONED_NAME.tar $SCRATCH_DIR/$VERSIONED_NAME-wlcs.tar
```

Comment:
Could this be a loop on the output from `git submodule`, so we don't have to do it again when we add a new submodule?